src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -44,6 +46,7 @@
   <ItemGroup>
     <PackageReference Include="Grpc.AspNetCore" />
     <PackageReference Include="KubernetesClient" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" />
```

Comment:
What is this for?

Reply:
The Microsoft.AspNetCore.Authorization.OpenIdConnectOptions that get in AppHpst’s DashbpardOptions.  I couldn’t figure out how to pass the whole object through to the context in the LifetimeHook though so I suppose it may make more sense to drop the dependency and just step supported properties manually.  I only mapped Authority, MetaAddress, ClientId, ClientSecret, and Scope

Reply:
Can we remove this and just make a poco

Reply:
I'll get it updated

Reply:
Updated